src/Traits/LivewireComponentParser.php

```
@@ -182,6 +286,24 @@ protected function getClassName()
 
     protected function getViewName()
     {
+        $prefix = $this->getModuleLowerName().'::';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$prefix'. (undefined)

(UnusedLocalVariable)